src/common/ui-components/HeroBanner/HeroBanner.tsx

```diff
@@ -31,6 +31,7 @@ const HeroBanner: React.FC<HeroBannerProps> = ({
           <div className={cx(styles.container)}>{children}</div>
         </section>
       </ImageWithCard>
+      <div className={classNames(styles.blur)}></div>
```

Comment:
> <div className={classNames(styles.blur)}></div>

Couldn't this instead be done with an inset box-shadow on `ImageWithCard`?

Reply:
Perhaps add it to `styles.topImgeWithCard`.

Reply:
@ahie ImageWithCard is used elsewhere, so the approach I tool reflects it.

Reply:
`topImageWithCard` is in `HeroBanner.module.scss`

Reply:
Do you think it can go as is for now? I mean, what is the rationale in here?

Reply:
Clean code. This will immediately cause a headache when making it responsive: you have to keep both the blur div and the `ImageWithCard` elements aligned.

Reply:
Ok, I can see your point. OK, Ill do it.

Reply:
@ahie I have tried to do what you were asking me and it looks like it might be a bit tricky to do considering the sensitive area: a lot of elements overlapping each other and topImageWithCard is being underneath the video element being the top one, the most sensible decision to me, at least, is this one so far. I believe this could be a tiny compromise that we make to have a good demo tomorrow, as it is clear that this component will change in the future.